01_variables/a_dumb_calculator.py

```diff
@@ -3,4 +3,16 @@
 # Then add the numbers together and divide by 2 
 # Display the result 
 # Your program must support numbers with decimals and not just integers 
-# for more info on this quiz, go to this url: http://www.programmr.com/dumb-calculator-5
\ No newline at end of file
+# for more info on this quiz, go to this url: http://www.programmr.com/dumb-calculator-5
+
+
+def simple_calc():
+    numbers = input("Enter three numbers separated with a space: ")
```

Comment:
I suggest that, you refactor to have the user be prompted three times, to make it easier to catch errors, e.g.

```python
print('please enter three numbers')
user_inputs = []
for i in range(3):
    u_i = input("Enter the first number: ")
    user_inputs.append(u_i)
```